cypress/e2e/po/lists/management.cattle.io.user.po.ts

```diff
@@ -6,7 +6,7 @@ import BaseResourceList from '@/cypress/e2e/po/lists/base-resource-list.po';
  */
 export default class MgmtUsersListPo extends BaseResourceList {
   create() {
-    return this.masthead().actions().eq(0).click();
+    return cy.get('[data-testid="masthead-create"]').click();
```

Comment:
We already use this same selector in `resource-list-masthead.po.ts`. 

Our `role.spec.ts` was sometimes failing to click the create button and I'm not entirely sure why. I noticed that the Refresh Group Memberships button pops in so I think that could be related.